pepper-apis/dsm-core/src/main/resources/liquibase/rgp/PEPPER-833_rgp_kit_display_name.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-833_rgp_kit_type_display_name" author="pegah">
+        <addColumn tableName="kit_type">
+            <column name="kit_display_name" type="VARCHAR(500)"></column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="PEPPER-833_rgp_blood" author="pegah">
+        <insert tableName="kit_type">
+            <column name="kit_type_name" value="BLOOD"></column>
```

Comment:
We have different blood kits for different studies in dev db, since every query has a matching with both realm and kit type name, so far DSM handles it ok because we didn't have a study with 2 blood kits. RGP would be a first and that's why I introduced searching by kit type ids instead

Reply:
What should be the unique key here?  Or should we add a UK to ddp_kit_request_settings to clarify that we do not support a ddp project that has multiple of the same named kit types?

Reply:
there is no unique key here (aside form the primary key of the table)
since in ddp_kit_request_settings, a realm is bundled together with a kit_type_id, I'm not sure how we can put a constraint on it to avoid pairing ddp_instance_id in that table with another value in kit_type table

Reply:
I think a unique key on (ddp_instance_id,kit_type_id) will work for ddp_kit_request.  These queries return the same results which to me says that the key won't break anything:

```
select concat(kit_type_id, '.', ddp_instance_id) from ddp_kit_request_settings

select distinct(concat(kit_type_id, '.', ddp_instance_id)) from ddp_kit_request_settings
```

Reply:
I think I'm confused,  since two kit entries in the `kit_type` table with different `kit_type_id`s could have the same kit_type_name which will pass the unique condition of `(ddp_instance_id,kit_type_id)` but would still be same kit type names